.changeset/tender-cameras-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bigcommerce/catalyst-core": patch
+---
+
+update create wishlist mutation
```

Comment:
🍹 Since this affects our changelog, let's be a bit more descriptive about _why_ we are changing it.